docs/guides/integration-prebid-mobile-summary.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Mobile Integration for Prebid Mobile SDK
+sidebar_label: Mobile Integration for Prebid Mobile SDK
+pagination_label: Mobile Integration for Prebid Mobile SDK
+description: Summary of information resources for setting up Prebid integration on mobile devices.
+hide_table_of_contents: false
+sidebar_position: 04
+---
+
+import Link from '@docusaurus/Link';
+
+# EUID Mobile Integration for Prebid Mobile SDK
+
+EUID integration with Prebid is also supported for Android mobile devices, using the Prebid Mobile SDK.
```

Comment:
ios too now?

Reply:
Thx @sunnywu and @dcaunt . Updated in 2 places.